cartoframes/context.py

```diff
@@ -148,6 +150,23 @@ def write(self, df, table_name, temp_dir='/tmp', overwrite=False,
         Returns:
             None
         """
+        if encode_geom:
+            is_geopandas = getattr(df, '_geometry_column_name')
```

Comment:
@ljwolf, this seems good, but I'm wondering if there's a way to do something more similar to:
```python
type(df) == geopandas.geodataframe.GeoDataFrame
```
But I can' think of a way to check that explicitly without importing geopandas, which I'd like to avoid to keep the dependencies slim.